docs/docsite/rst/network/getting_started/first_playbook.rst

```diff
@@ -86,22 +86,21 @@ The playbook contains one play with two tasks, and should generate output like t
 .. code-block:: bash
 
    $ ansible-playbook -i vyos.example.net, -u ansible -k -e ansible_network_os=vyos first_playbook.yml
-   
+
    PLAY [First Playbook]
    ***************************************************************************************************************************
-   
+
    TASK [Gathering Facts]
```

Comment:
Does the sample playbook still generate this output with `gather_facts` set to `No`?